components/Chat/Messages.tsx

```
@@ -1,19 +1,43 @@
+                  <TvMinimalPlay size={32} color="#000000" />
+                )}
+              </div>
+            )}
+            <div
+              className={`p-3 rounded-lg ${
+                message.role === "user"
+                  ? "bg-black text-white max-w-1/2 float-right"
+                  : "bg-transparent text-black w-[90%]"
+              }`}
+            >
+              <p className="text-sm" dangerouslySetInnerHTML={{ __html: marked.parse(message.toolInvocations?.[0]?.result ?? message.content, { async: false }) as string }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)